### PR TITLE
Added links to writing guides and aids for non-writers.

### PR DESCRIPTION
**Pull request by rimutaka**
I added a section with links to resources of value to programmers to help them learn the basics of technical writing.

### CODE REVIEW

File: writing-tips.md [hunk 1]
```diff
@@ -55,3 +55,19 @@ Here are some title examples and their suggested file name structures:
 ## Definitions
 
 If you are repeatedly using terms throughout a large article, you can define them once and then use them repeatedly in the body of the article.
+
+## Writing help for programmers
+These resources will make you better at explaining your products and ideas. 
+
+* [Google Technical Writing Course](https://developers.google.com/tech-writing)
+  > A 1 hour long self-paced course. It is a must-read for every programmer.
```
Comment (jaredmorgs) [inline writing-tips.md:63]: Suggest changing to:

An hour-long...

Also change any other numbers 0 - 9 from numerals to words in this PR in line with the Google style guide.

https://developers.google.com/style/numbers#numbers-as-words

**Comment by mgan59** [comment]
Feel like `writing-tips` might serve our audience best as a set of actual tips...  And maybe we build out a separate topic for `writing-resources.md` and break down various sections for different audiences -- developers learning to write, tech-comm podcasts, etc....

**Comment by jaredmorgs** [comment]
> Feel like `writing-tips` might serve our audience best as a set of actual tips... And maybe we build out a separate topic for `writing-resources.md` and break down various sections for different audiences -- developers learning to write, tech-comm podcasts, etc....

That's probably a good observation. These are more resources than tips. There are tips about filling in templates in some of the template repos.

**Comment by rimutaka** [comment]
Do we even need links to those resources?

**Comment by emckean** [comment]
+1 for splitting this out into "Writing Resources" and "Writing Tips" and having "Writing Tips" be very, very short :)

**Comment by rimutaka** [comment]
I'll split and resubmit.

**Comment by camerons** [comment]
Hi @rimutaka. This is our oldest pull request, so will be talked about in our weekly status meeting in 10 hours. Would you mind letting us know where you are up to with this.

**Comment by rimutaka** [comment]
Cameron, I didn't get a chance to look into it. These are just the links that were floated in Slack. I'm not sure there is much value in them. Probably not even worth discussing on the call :)

**Pull request closed without merging**